src/main/kotlin/com/chloe/kotlinserv/route/GetCountryStatsRoute.kt

```diff
@@ -54,7 +62,7 @@ class GetCountryStatsRoute @Inject constructor(
             }
         } catch (e: IllegalArgumentException) {
```

Comment:
Add one more catch for Throwable to handle all other errors. Respond with 500 code